consumer/hikey/installation/board-recovery.md

```diff
@@ -13,37 +13,26 @@ For most users a board can be “recovered” from a software failure by reloadi
 
 #### Choose your Operating System and download the following files
 
-Build Folders <a href="http://builds.96boards.org/releases/reference-platform/debian/hikey/16.06/bootloader/" target="_blank">Debian</a>
+Build Folders [UEFI](http://builds.96boards.org/snapshots/reference-platform/components/uefi-staging/latest/hikey/debug/)
 
```

Comment:
release?

Reply:
I've personally tested on debug. If it seems ok, I can devide the download section into two, one for release and one for debug that way user can download the one they like. Let me know if this seems ok